### PR TITLE
fix: Added more detail logging and fixed some issues with API

### LINKED ISSUE #50: Update logging
Add more detailed logs in application

### PR DESCRIPTION
**Pull request by karelumair**
- Added more detail logging.
- Fixed issue with get quotes by tags filter.

closes #50

### CODE REVIEW

**Comment by karelumair** [comment]
Should I merge this now?

File: resources/author.py [hunk 1]
```diff
@@ -18,9 +18,13 @@ def get(self) -> Response:
...
             Response: JSON object of all the Authors
         """
+        current_app.logger.info("GET Authors - REQUEST RECEIVED")
+
         cursor = Author.objects().exclude("scrapeId")
         authors = cursor_to_json(cursor)
-        current_app.logger.info("GET Authors")
+        n_authors = len(authors)
+
+        current_app.logger.info(f"GET Authors - FETCHED {n_authors} Authors")
```
Comment (pratikGhodke1) [inline resources/author.py]:
```suggestion
        current_app.logger.info(f"GET Authors - FETCHED {len(authors)} Authors")
```

File: resources/quote.py [hunk 2]
```diff
@@ -6,7 +6,7 @@
 from flask_restful import Resource
 from database.models import Quote
 from database.schemas import QuoteSchema, QuoteUpdateSchema
-from utils.utils import cursor_to_json
+from utils.utils import cursor_to_json, quote_aggregate_to_json
```
Comment (pratikGhodke1) [inline resources/quote.py:9]: Remove this

Comment (pratikGhodke1) [inline resources/quote.py:9]:
```suggestion
from utils.utils import cursor_to_json
```

**Review by pratikGhodke1 — APPROVED** [review]
Looks good!

**Comment by pratikGhodke1** [comment]
You could have updated the first commit with `len(n_authors)` changes instead of making new commit using `git commit --amend`

**Pull request merged**